e2e/volume_ops_test.go

```diff
@@ -112,27 +111,95 @@ func testVolumeCreate(t *testing.T) {
 	createReq.Name = "##@@#@!#@!!@#"
 	_, err = client.VolumeCreate(createReq)
 	r.NotNil(err)
+
 }
 
+func testVolumeCreateWithFlags(t *testing.T) {
+	r := require.New(t)
+
+	var brickPaths []string
+
+	for i := 1; i <= 4; i++ {
+		brickPaths = append(brickPaths, fmt.Sprintf(baseWorkdir+t.Name()+"/b/%d", i))
+	}
+
+	flags := make(map[string]bool)
+	//set flags to allow rootdir
+	flags["allow-root-dir"] = true
+	//set flags create brick dir
+	flags["create-brick-dir"] = true
+
+	createReqBrick := api.VolCreateReq{
+		Name: t.Name(),
+		Subvols: []api.SubvolReq{
+			{
+				ReplicaCount: 2,
+				Type:         "replicate",
+				Bricks: []api.BrickReq{
+					{PeerID: gds[0].PeerID(), Path: brickPaths[0]},
+					{PeerID: gds[1].PeerID(), Path: brickPaths[1]},
+				},
+			},
+			{
+				Type:         "replicate",
+				ReplicaCount: 2,
+				Bricks: []api.BrickReq{
+					{PeerID: gds[0].PeerID(), Path: brickPaths[2]},
+					{PeerID: gds[1].PeerID(), Path: brickPaths[3]},
+				},
+			},
+		},
+		Flags: flags,
+	}
+
+	_, err := client.VolumeCreate(createReqBrick)
+	r.Nil(err)
+
+	//delete volume
+	r.Nil(client.VolumeDelete(t.Name()))
+
+	createReqBrick.Name = "mountbrick"
```

Comment:
Same here, base the volume name based on the test name using `t.Name()`

Reply:
done